File: cpp/raindrops/raindrops.cpp
#include "raindrops.h"

namespace raindrops {

    std::string convert(int number){

        std::string output = "";

        if(number % 3 == 0){
            output += "Pling";
        }

        if(number % 5 == 0){
            output += "Plang";
        }

        if(number % 7 == 0){
            output += "Plong";
        }

        if(output == ""){
            output += std::to_string(number);
        }

        return output;

    }

}  // namespace raindrops

File: typescript/raindrops/raindrops.ts
class Raindrops {
    convert(drops: number): string {
        let output = `${drops % 3 == 0 ? 'Pling' : ''}${drops % 5 == 0 ? 'Plang' : ''}${drops % 7 == 0 ? 'Plong' : ''}`;
        return output.length !== 0 ? output : drops.toString() ;
    }
  }
  
  export default Raindrops
  